unlock-app/src/__tests__/config.test.js

```diff
@@ -146,7 +146,7 @@ describe('config', () => {
     it('should contain the right URLs for chain explorers', () => {
       expect.assertions(2)
       expect(Object.keys(config.chainExplorerUrlBuilders)).toHaveLength(1)
-      expect(config.chainExplorerUrlBuilders.etherScan('0x0')).toEqual(false)
+      expect(config.chainExplorerUrlBuilders.etherScan('0x0')).toEqual('')
```

Comment:
let's use `toBe`

Reply:
I think if that's an important change to be made, it should be done to that file as a whole in a separate PR instead of introducing it to just one of the many test cases using `toEqual` in this PR.

Reply:
that makes sense, I did not see the wider context of the needed change. Open an issue?